docs/hardware.rst

```diff
@@ -424,6 +424,11 @@ after disconnecting them.
 
 .. _nuc11_recommendation:
 
+Because of the newer hardware and the drivers required, you will need to use a
```

Comment:
Should this info be flagged in the install doc as well/instead? Something along the lines of a note just before the Ubuntu install recommending the use of the HWE kernel for NUC12s specifically and newer hardware in general.

Reply:
That's a good idea. I think it makes sense to flag this specifically in the NUC12 section of the hardware guide since we know that to be needed, but I'll add a general note in the install doc as well.